viam-cartographer/src/slam_service/slam_service_test.cc

```diff
@@ -318,32 +318,6 @@ BOOST_AUTO_TEST_CASE(SetActionMode_invalid_case) {
     utils::removeTmpDirectory(tmp_dir);
 }
 
-BOOST_AUTO_TEST_CASE(GetNextDataFileOffline_not_enough_data) {
```

Comment:
I ended up removing this test. We still will not try to construct a map with just two files, but instead we'll just keep looping until we find more files. I couldn't figure out how to test this since `GetNextDataFileOffline()` will just block.

Reply:
Could we make the test check that if there are less than 2 files, "No new data found" is logged?